crates/prover-bin/src/zk_circuits_handler/universal.rs

```diff
@@ -3,27 +3,34 @@ use std::path::Path;
 use super::CircuitsHandler;
 use async_trait::async_trait;
 use eyre::Result;
-use scroll_proving_sdk::prover::ProofType;
+use libzkp::ProvintTaskExt;
 use scroll_zkvm_prover::{Prover, ProverConfig};
 use scroll_zkvm_types::ProvingTask;
 use tokio::sync::Mutex;
 pub struct UniversalHandler {
     prover: Prover,
 }
 
+// additional config dispatched with proving task
+#[derive(Debug, Default)]
+pub(crate) struct TaskConfig {
```

Comment:
very wierd, I think ProvintTaskExt is enough, don't need to have a new TaskConfig